start_vscode_stanage.sh

```diff
@@ -257,7 +282,7 @@ else
 fi
 
 # set modules
-VSC_MODULE_COMMAND="apps/vscode-server/4.2.0/binary dev/git/2.35.2/gcc-4.9.4"
+VSC_MODULE_COMMAND="code-server/4.16.1 git/2.39.2-GCCcore-12.2.0-nodocs"
```

Comment:
This version of Git does not appear to exist for the GPU nodes, but they do have ``git/2.41.0-GCCcore-12.3.0-nodocs``. Can we get parity around ``git/2.41.0-GCCcore-12.3.0-nodocs`` on all nodes please. 

We are also missing code-server, from the GPU nodes (I am adding this now.)

Likewise, a general docs update will be needed for both modules.

 In a more general sense, we may wish to install these modules within the common area rather than arch specific areas, or at least should make them available for GPUs.
 
 
 For now suggesting we do not target specific versions so latest is used.
 
```suggestion
VSC_MODULE_COMMAND="code-server git"
```